src/sentry/options/defaults.py

```diff
@@ -185,3 +185,8 @@
 
 # Normalization after processors
 register('store.normalize-after-processing', default=0.0)
+
+# Post Process Error Hook Sampling
+register('post-process.use-error-hook-sampling', default=False)
```

Comment:
@JTCunning / @tkaemming couldn't think of better names so open to others if this could be improved upon